actions/publish_message_MOD.js

```diff
@@ -0,0 +1,73 @@
+module.exports = {
+  name: 'Announce/Publish Message',
+  section: 'Messaging',
+
+  subtitle (data) {
+    const message = ['Command Message', 'Temp Variable', 'Server Variable', 'Global Variable']
+    return `${message[parseInt(data.message)]}`
+  },
+
+  variableStorage (data, varType) {
+    const type = parseInt(data.storage)
+    if (type !== varType) return
+    return ([data.varName2, 'Message'])
```

Comment:
To be consistent with the other mods:
```suggestion
    if (parseInt(data.storage) !== varType) return
    return [data.varName2, 'Message']
```